src/components/FMenu/FMenuButton.vue

```diff
@@ -1,43 +1,193 @@
 <template>
-  <f-button
-    class="Fmenu-side__button"
-    :icon="iconMenu"
-    :bigger="true"
-    :radius="false"
-    :color="color"
-    @click="handler"
-  />
+  <div class="FMenuButton" :class="arrowMenu">
+    <div class="FMenuButton__container" @click="main">
+      <div class="FMenuButton__icon">
+        <div class="FMenuButton__line"></div>
```

Comment:
coloca as três linhas como um v-for.
ref: https://br.vuejs.org/v2/guide/list.html